python/private/toolchains_repo.bzl

```diff
@@ -236,14 +237,15 @@ actions.""",
 )
 
 def _host_toolchain_impl(rctx):
+    logger = repo_utils.logger(rctx)
     rctx.file("BUILD.bazel", """\
 # Generated by python/private/toolchains_repo.bzl
 
 exports_files(["python"], visibility = ["//visibility:public"])
 """)
 
-    (os_name, arch) = get_host_os_arch(rctx)
-    host_platform = get_host_platform(os_name, arch)
+    (os_name, arch) = _get_host_os_arch(rctx, logger)
```

Comment:
If I remember correctly this code was written before rctx and mctx had the architecture. I think we could look into removing the dependency on uname here.

And just use the repo utils equivalents.

Reply:
Ahh, that would explain a bit. I always wondered why we were calling uname when rctx seemed to have the equivalent.